pandas/tests/arithmetic/test_numeric.py

```diff
@@ -1284,3 +1287,35 @@ def test_dataframe_div_silenced():
     )
     with tm.assert_produces_warning(None):
         pdf1.div(pdf2, fill_value=0)
+
+
+class TestNumericArraylikeArithmeticWithBool:
+    @pytest.mark.skipif(
+        compat.is_platform_32bit(),
+        reason="Arithmetic operations are not supported on 32bit",
+    )
+    @pytest.mark.parametrize("num", [1, 1.0])
+    def test_array_like_bool_and_num_op_coerce(
+        self, num, all_arithmetic_functions, box_with_array
+    ):
+        # https://github.com/pandas-dev/pandas/issues/18549
+        op = all_arithmetic_functions
+
+        if op.__name__ in [
+            "floordiv",
+            "mod",
+            "mul",
+            "pow",
+            "rfloordiv",
+            "rpow",
+            "rmod",
+            "rmul",
+            "rtruediv",
+            "truediv",
+        ]:
+            pytest.skip("Arithmetic operation is not supported")
```

Comment:
Probably better to xfail than skip here; don’t think it matters when used imperatively but former would be more fitting

Reply:
is this something that _should_ be supported but isnt?

Reply:
> is this something that _should_ be supported but isnt?

I am not sure.